src/core/shelf.sol

```diff
@@ -17,61 +17,61 @@
 pragma solidity >=0.4.24;
 pragma experimental ABIEncoderV2;
 
-import { TitleOwned } from "tinlake-title/title.sol";
-
-contract AppraiserLike {
-    function appraise(uint, address, uint) public returns (uint);
-}
+import "ds-note/note.sol";
+import { Title, TitleOwned } from "tinlake-title/title.sol";
```

Comment:
```suggestion
import { TitleLike, TitleOwned } from "tinlake-title/title.sol";
```

Reply:
TitleLike needs to be extended with issue & close function signatures first

Reply:
Why not do that now?